merlin/models/utils/dataset.py

```diff
@@ -52,16 +52,34 @@ def dataset_to_coo(dataset: Dataset):
 
 def unique_rows_by_features(
     dataset: Dataset, features_tag: Union[str, Tags], grouping_tag: Union[str, Tags]
```

Comment:
Would it make sense to make `Tags.ID` the default for `grouping_tag`?

Reply:
yes, I think that would be good if that's the main use-case we have for this helper.